### PR TITLE
mark events for performance monitoring

### PR DESCRIPTION
**Pull request by gilluminate**
### Description Of Changes

mark all FidesJS events for performance monitoring

### CODE REVIEW

File: clients/fides-js/src/lib/events.ts [hunk 1]
```diff
@@ -65,6 +65,7 @@ export const dispatchFidesEvent = (
     const event = new CustomEvent(type, {
       detail: { ...cookie, debug, extraDetails: constructedExtraDetails },
     });
+    performance.mark(type);
```
Comment (NevilleS) [inline clients/fides-js/src/lib/events.ts:68]: nit:
```suggestion
    performance && performance.mark(type);
```

I know caniuse says `performance` is everywhere, but it's probably something that could be disabled by a useragent, etc?

**Review by NevilleS — CHANGES_REQUESTED** [review]
One nitpick here!

File: clients/fides-js/src/lib/events.ts (hunk 1 above)
Comment (gilluminate) [reply clients/fides-js/src/lib/events.ts:68]: gooooood call!

**Review by NevilleS — APPROVED** [review]
ship it!

**Pull request merged**